src/Cli/dotnet/Commands/Test/MTP/TestApplicationHandler.cs

```diff
@@ -140,12 +152,15 @@ internal void OnTestResultsReceived(TestResultMessages testResultMessage)
                 standardOutput: testResult.StandardOutput,
                 errorOutput: testResult.ErrorOutput);
         }
-
-        LogTestResults(testResultMessage);
     }
 
     internal void OnFileArtifactsReceived(FileArtifactMessages fileArtifactMessages)
     {
+        LogFileArtifacts(fileArtifactMessages);
```

Comment:
what will the log look like in case of _options.IsHelp? it would be good to have the log make it clear that the message was ignored, triggered an exception in that case

Reply:
sure, will the log look something like:

```
FileArtifacts: UID: x
FileArtifacts: Other Properties
[Error] FileArtifacts was sent in --help mode which is invalid
```

As in, will the log be sufficient for us to investigate why there were problems processing the file artifact?